contrib/jmx-metrics/jmx-metrics.gradle

```diff
@@ -0,0 +1,79 @@
+plugins {
+    id "com.github.johnrengelman.shadow" version "6.0.0"
+}
+
+apply from: project.contrib
+apply plugin: 'com.github.johnrengelman.shadow'
+
+archivesBaseName = 'io.opentelemetry.contrib.jmx-metrics'
+
+description = 'JMX metrics gathering Groovy script runner'
+
+// we are joint compiling so rely on groovy plugin entirely
+sourceSets.main.groovy.srcDirs = [
+    "src/main/java",
+    "src/main/groovy"
+]
+sourceSets.test.groovy.srcDirs = [
+    "src/test/java",
+    "src/test/groovy"
+]
+sourceSets.main.java.srcDirs = []
+sourceSets.test.java.srcDirs = []
+
+jar {
+    manifest {
+        attributes('Main-Class': 'io.opentelemetry.contrib.jmxmetrics.JmxMetrics')
```

Comment:
I think instead we can add the application plugin to also have run task right?

Similarly, since this is an application, not an SDK, no reason to leave it Java 7, Java 8 should be good, or even Java 11

Reply:
Updated w/ application plugin.

As to java version, would it be ok to defer this decision w/ a tracking issue?